content/influxdb/cloud-dedicated/get-started/write.md

```diff
@@ -266,13 +334,14 @@ The following example uses cURL and the InfluxDB v1 API to write line protocol
 to InfluxDB:
 
 {{% code-placeholders "DATABASE_TOKEN" %}}
-{{% influxdb/custom-timestamps %}}
+
 ```sh
-curl -i 'https://{{< influxdb/host >}}/write?db=get-started&precision=s' \
-    --header 'Authorization: Bearer DATABASE_TOKEN' \
-    --header "Content-type: text/plain; charset=utf-8" \
-    --header "Accept: application/json" \
-    --data-binary "
+curl --silent -w "%{http_code}\n" \
```

Comment:
The [cURL write out docs](https://everything.curl.dev/usingcurl/verbose/writeout#available-write-out-variables), recommend `response_code` over `http_code`.

One concern I have with this is that it only returns the response code. While users shouldn't have any issues if the follow the tutorial, there is a chance the request could fail and more information would be helpful. Should this also output the `errormsg`? (This is left blank if there is no error).

```suggestion
curl --silent -w "%{response_code}: %{errormsg}\n" \
```